poetry.lock

```diff
@@ -1628,13 +1628,13 @@ devenv = ["check-manifest", "pytest (>=4.3)", "pytest-cov", "pytest-mock (>=3.3)
 
 [[package]]
 name = "urllib3"
-version = "2.2.1"
+version = "2.2.2"
 description = "HTTP library with thread-safe connection pooling, file post, and more."
-optional = true
+optional = false
```

Comment:
Dependabot changed optional to false again.

https://github.com/dependabot/dependabot-core/issues/3104

Reply:
Fixed by manually editing poetry.lock